app/controllers/ValidationController.scala

```diff
@@ -53,11 +55,13 @@ class ValidationController @Inject() (implicit val env: Environment[User, Sessio
 
     request.identity match {
       case Some(user) =>
+        val cityStr: String = Play.configuration.getString("city-id").get
         val validationData = getDataForValidationPages(user, ipAddress, labelCount = 10, "Visit_Validate")
+        val cityName: String = Play.configuration.getString("city-params.city-name." + cityStr).get
```

Comment:
Instead of passing along the city name, can you actually add a new config to `cityparams.conf`? It'd probably be best to deal with things like which city we are looking at back in the config files, and try to make the actual code as focused as possible on the actual functionality :)